packages/config-eslint/moss-lint-plugin/rules/only-valid-token-names.js

```diff
@@ -0,0 +1,109 @@
+// FIXME: temporary solution with test.json file
+import themeInterface from "../test.json" with { type: "json" };
+
+const colorKeysToCssValues = (colors) => {
+  const keys = Object.keys(colors);
+  return keys.map((color) => "--color-" + color.replaceAll(".", "-"));
+};
+
+const VALID_TOKENS = colorKeysToCssValues(themeInterface.colors);
```

Comment:
We might consider to use a Set for `VALID_TOKENS`: Converting `VALID_TOKENS` from an array to a Set will improve lookup efficiency from O(n) to O(1)


```typescript
const VALID_TOKENS = new Set(colorKeysToCssValues(themeInterface.colors));
```